net/netatalk/Makefile

```diff
@@ -8,81 +8,103 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=netatalk
-PKG_VERSION:=3.2.5
+PKG_VERSION:=4.0.3
 PKG_RELEASE:=1
-PKG_LICENSE:=GPL-2.0-or-later
 
-#PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=@SF/netatalk
-PKG_HASH:=57de9a7ed411029d6176e429a14ef314460251d8aebe7139aeadc35633d9584b
+PKG_HASH:=beaf77962fe15deffb5a09853b2f1a07c308f8bf27369398d62b46d9be7ce83f
 
-PKG_BUILD_PARALLEL:=1
-PKG_INSTALL:=1
-PKG_FIXUP:=autoreconf
+# Unicode DB file required for compilation
+UC_DATA_FILE:=UnicodeData.txt
+UC_DATA_URL:=https://www.unicode.org/Public/16.0.0/ucd/
+UC_DATA_HASH:=ff58e5823bd095166564a006e47d111130813dcf8bf234ef79fa51a870edb48f
 
+PKG_MAINTAINER:=Antonio Pastor <antonio.pastor@gmail.com>
+PKG_LICENSE:=GPL-2.0-or-later
+PKG_LICENSE_FILES:=COPYING
 PKG_CPE_ID:=cpe:/a:netatalk:netatalk
 
-PKG_BUILD_DEPENDS:=libevent2
-
 include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/nls.mk
+include $(INCLUDE_DIR)/meson.mk
+
+PKG_BUILD_PARALLEL:=1
+PKG_INSTALL:=0
```

Comment:
Why do we need these two variables with meson? :)

```suggestion
```

Reply:
Yes. Will do.

Reply:
Well, `PKG_BUILD_PARALLEL:=1` should stay. If not `include\package.mk` will just make singlethreaded.

Reply:
Tested with & without `PKG_BUILD_PARALLEL` and saw no difference. This package is so small it wouldn't matter anyways. Removed.

Reply:
No, no. Thats not correct. Your package is using meson and ninja. While checking CI log, you can see it there:
> 2024-10-29T02:57:34.4488563Z MAKEFLAGS="--jobserver-auth=3,4" /builder/staging_dir/host/bin/ninja   -C /builder/build_dir/target-arm_cortex-a9+vfpv3-d16_musl_eabi/netatalk-4.0.3/openwrt-build 
2024-10-29T02:57:34.4546074Z ninja: Entering directory `/builder/build_dir/target-arm_cortex-a9+vfpv3-d16_musl_eabi/netatalk-4.0.3/openwrt-build'

Also, if you look at this commit: https://github.com/openwrt/openwrt/commit/97258f53634d7237a2962aec3387f011047ce83b, if the package uses ninja, then it is using PKG_BUILD_PARALLEL. 

Also, in the CI/CI log, you can see that there is also cmake for netatalk.